translations/es.ts

```diff
@@ -1215,7 +1215,7 @@ serán enviados cuando los veas aparecer de nuevo en línea.</translation>
     <message>
         <location filename="../src/widget/form/settings/generalsettings.ui" line="521"/>
         <source>Timestamp format:</source>
-        <translation>Formato de marca temporal:</translation>
+        <translation>Formato de marcas temporales:</translation>
```

Comment:
creo que yo estaba borracho cuando lo escribí. Se nota en "la mismo opinión".
NO, date is a separate one. so We have date format and timestamp format. All is fine.
I lumped them together in my head last night, and thought the timestamp format were the date thingy, but it ain't. I blame the alcohol. So everything is ok with the original.
Sorry for the confusion.

Ahora bien, si tenés "formato de LA fecha", por qué no también "formaot de LA marca temporal"
Yo elegí el plural porque fecha la tenés solo una cada día, pero muchas marcas temporales. 
Claro, puedes argumentar que es solo una por mensaje, o que hablas en general ("la marca temporal sirve para..." Pero también lo general puedes expresar con el plural ("marcas temporales sirven..."
Bueno, no sé. Ya me marreé con tanta cosa.